### DISCUSSION TITLE
storeToRefs makes reactive -> ref

### DISCUSSION THREAD

**Discussion by aeropop** [vuejs/pinia#2601] [Help and Questions] [answered]
storeToRefs turns 'reactive' -> 'ref'. I use reactive mostly to not write '.value' every time. Any way to keep this functionality?
// store.ts
```
const useStore = defineStore('store', () => {
  const state = reactive({
    some: false
  })

  return {
    state
  }
})
```
// component.vue

```
const { state } = storeToRefs(useStore());
console.log(state.value.some) // false;
console.log(state.some) // undefined <- 😢 
```

**Comment by posva** [comment]
You don't need `storeToRefs()` unless you pass individual state around. so you can write

```ts
const store = useStore()
store.state.value
```

**Comment by aeropop** [comment, marked as answer]
I did want to pass individual state around. But you're right `store.state.some` is not too verbose either. Thanks!
